modules/cluster-autoscaler/variables.tf

```diff
@@ -10,8 +10,7 @@ variable "helm" {
   type        = map
   default = {
     name            = "eks-as"
-    repository      = "https://kubernetes.github.io/autoscaler"
-    chart           = "cluster-autoscaler-chart"
```

Comment:
Still users can replace the helm chart with something else even after this PR is merged.